src/main/webapp/app/overview/course-exercises/course-exercises.component.html

```diff
@@ -1,48 +1,96 @@
+                        [translateValues]="{total: weeklyExercisesGrouped[weekKey].exercises.length}">
+                        Exercises: {{weeklyExercisesGrouped[weekKey].exercises.length}}
+                    </span>
+                </div>
+                <div *ngIf="!weeklyExercisesGrouped[weekKey].isCollapsed">
+                    <jhi-course-exercise-row class="pb-1" [exercise]="exercise" [course]="course" *ngFor="let exercise of weeklyExercisesGrouped[weekKey].exercises"></jhi-course-exercise-row>
+                </div>
+                <div class="collapsed"></div>
+            </div>
+            <div *ngIf="course.exercises.length === 0">
+                <div class="row">
+                    <div class="col">
+                        {{"arTeMiSApp.courseOverview.exerciseList.noExercises" | translate}}
+                    </div>
+                </div>
             </div>
         </div>
     </div>
-    <div class="exercise-row-container mb-3" *ngFor="let weekKey of weeklyIndexKeys">
-        <div class="control-label" [ngClass]="weeklyExercisesGrouped[weekKey].isCurrentWeek ? 'text-primary': ''"
-             (click)="weeklyExercisesGrouped[weekKey].isCollapsed = !weeklyExercisesGrouped[weekKey].isCollapsed">
-            <fa-icon class="pr-3"
-                     [icon]="weeklyExercisesGrouped[weekKey].isCollapsed ? 'angle-down' : 'angle-up'"></fa-icon>
-            <span [innerHtml]="weeklyExercisesGrouped[weekKey].label"></span>
-            <span
-                class="ml-2"
-                jhiTranslate="arTeMiSApp.courseOverview.exerciseList.exerciseGroupHeader"
-                [translateValues]="{total: weeklyExercisesGrouped[weekKey].exercises.length}">
-                Exercises: {{weeklyExercisesGrouped[weekKey].exercises.length}}
-            </span>
-        </div>
-        <div *ngIf="!weeklyExercisesGrouped[weekKey].isCollapsed">
-            <jhi-course-exercise-row
-                class="pb-1"
-                [exercise]="exercise"
-                [course]="course"
-                *ngFor="let exercise of weeklyExercisesGrouped[weekKey].exercises"></jhi-course-exercise-row>
-        </div>
-        <div class="collapsed"></div>
-    </div>
-    <div *ngIf="course.exercises.length === 0">
-        <div class="row">
-            <div class="col">
-                {{"arTeMiSApp.courseOverview.exerciseList.noExercises" | translate}}
+    <div class="col-12 col-md-4 course-information">
+        <jhi-side-panel [panelHeader]="('arTeMiSApp.courseOverview.exerciseList.details.courseInformation' | translate)">
+            <div class="row mb-1">
+                <div class="col-8">{{"arTeMiSApp.courseOverview.exerciseList.details.totalExercises" | translate}}</div>
+                <div class="col-4">{{course.exercises.length}}</div>
+            </div>
+            <div class="row mb-1">
+                <div class="col-8">{{"arTeMiSApp.courseOverview.exerciseList.details.totalQuizExercises" | translate}}</div>
```

Comment:
Can you use a for loop and go through all exercise types stored in a map? (see other comment).
This would avoid code duplication here